### PR TITLE
Extend CONTRIBUTORS.md with security researchers

### PR DESCRIPTION
**Pull request by joestringer**
We appreciate the efforts of these community members who have been
researching the security posture of Cilium and raising discussions
with the project in order to improve Cilium's security footprint.

### CODE REVIEW

**Comment by xmulligan** [comment]
This is great, do they want their emails published though?

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -10,3 +10,15 @@ Please either submit a pull request adding your name, or mention it to another o
 and we will add you. **All** contributors belong here. 💯
 
 * [Bill Mulligan](https://github.com/xmulligan)
+* [Daniel Iziourov](https://github.com/danmx)
+* [Daniel Teixeira](https://github.com/daniel-f3)
+* Jietao Xiao (jietaoXiao@stu.xidian.edu.cn)
+* Jinku Li (jkli@xidian.edu.cn)
+* [Micah Hausler](https://github.com/micahhausler)
+* Nanzi Yang (nzyang@stu.xidian.edu.cn)
```
Comment (joestringer) [inline CONTRIBUTORS.md:18]: @younaman can you confirm whether it is OK to publish you and your colleagues' emails like this?

Comment (joestringer) [reply CONTRIBUTORS.md:18]: The alternative is that I can link your name here to your github profile, and just keep the names without emails for the others you are working with. I don't mind which way to present these, let us know what you think?

Comment (younaman) [reply CONTRIBUTORS.md:18]: It's OK, thanks! I just want my name and e-mail like this.

**Review by younaman — APPROVED** [review]
It's just what I want! Thanks:) Have a nice day!

File: CONTRIBUTORS.md (hunk 1 above)
Comment (younaman) [reply CONTRIBUTORS.md:18]: I am not familiar with the Github's "approve changes", and I am sorry for the delay:) What you added is what I want, thank you again for your patience and work @joestringer.
Regards,
Nanzi Yang.

Comment (joestringer) [reply CONTRIBUTORS.md:18]: It's no trouble, thanks!

**Review by xmulligan — APPROVED** [review]

**Pull request merged**